utils/installer/install.sh

```diff
@@ -1,4 +1,12 @@
 #!/usr/bin/env bash
+#/ Usage: install.sh [<options>]
```

Comment:
It's easier and more standard if you make this into a function `usage()`.

Reply:
I will move it into a `usage` funciton.